coop_shift/__manifest__.py

```diff
@@ -1,23 +1,24 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
 # @author Cyril GASPARD
 # @author Julien WESTE
 # @author Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Coop Shift',
-    'version': '9.0.7.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Tools',
-    'author': 'Julien WESTE, Sylvain LE GAL, Cyril Gaspard, La Louve',
+    'author': 'Julien WESTE, Sylvain LE GAL, Cyril Gaspard, La Louve, Druidoo',
     'website': 'http://www.lalouve.net',
     'license': 'AGPL-3',
     'depends': [
         'base',
         'event',
         'event_sale',
         'mail',
-        'barcodes_generate',
+        'barcodes_generator_partner',
+        'barcodes_generator_product',
         'connector',
```

Comment:
Please check my comments about the `connector` dependancy on the spreadsheet file.
In 12.0, this module was separated into `connector` and `queue_job`.

I'm pretty sure we only need `queue_job` now, and we can replace this dependancy.